client/src/components/App.js

```diff
@@ -21,11 +21,16 @@ const makeUserSession = () => {
 };
 
 class App extends Component {
+
   constructor(props) {
     super(props);
     this.userSession = new UserSession({ appConfig });
   }
 
+ popup = setTimeout(function() { alert("Fill in your health survey for today!") }, 5000)
```

Comment:
Will this happy every time the app re-renders? I feel like a notification every 5 seconds or every time it re-renders would be too much? Or am I missing context?

Reply:
Ok so I did git rebase master, hopefully it is good now. The notification is supposed to show up 5 seconds after you open the app. I was just thinking of this issue too. The function may need some specification stating that if the survey is filled out, then don't show this notification....

Reply:
That makes sense. Looks like there's still merge conflicts. Did you push after rebasing?

Reply:
I forgot to push 😣sorry about that! It should be fine now